src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -165,37 +159,60 @@ public ProcessResult updateCase(
         } catch (UnprocessableEntity exception) {
             ClientServiceErrorResponse errorResponse = serviceResponseParser.parseResponseBody(exception);
             return new ProcessResult(errorResponse.warnings, errorResponse.errors);
+        } catch (FeignException.UnprocessableEntity exception) {
+            String msg = getErrorMessage(configItem.getService(), existingCaseId, exceptionRecord.id)
+                + " - CCD could not process request";
+            // is there a better way?
+            boolean isStartEvent = exception.request().url().contains("event-triggers");
```

Comment:
This code is duplicated,  it should be in a method

Reply:
small steps. this portion is not nice to begin with. but goal is to remove try/catch from submit event. whatever it takes